diff --git a/Release/src/CMakeLists.txt b/Release/src/CMakeLists.txt
index a822d78..438d3d5 100644
--- a/Release/src/CMakeLists.txt
+++ b/Release/src/CMakeLists.txt
@@ -50,7 +50,7 @@ if(UNIX)
   else()
     list(APPEND SOURCES pplx/pplxlinux.cpp)
   endif()
-  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${WARNINGS} -Werror -pedantic")
+  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${WARNINGS} -pedantic")
 elseif(WIN32)
   set(SOURCES
     ${SOURCES_COMMON}
